### ISSUE TITLE
After creating a QR code the "Generate QR Code" button is disabled. Here, if the user is barred from creating another QR code using another text / url until they download it.

### ISSUE THREAD

**Issue by onepointpb** [OnePointGlobal/OnePoint-Global-Account-Toolkit#176]
Scenario:
1. Enter Text / URL to generate QR code.
2. Click "Generate QR Code".
3. Now "Generate QR Code" button is disabled until the user downloads it or refresh the page or click on the tab. 
This approach is not correct as the user may like to scan the QR code using their mobile device instead of downloading it.

**Comment by deepika-Gopala** [comment]
they can still scan the code as long as the code is visible , if you mean that they change their mind and want to change the url for that we will enable the button.

**Comment by onepointpb** [comment]
Yes, if they change their mind and type some other url / text then make this button enabled.

**Comment by onepointpb** [comment]
Also, change the downloaded file name to "QR Code" instead of "Chart".

**Comment by onepointpb** [comment]
The name of the file is not changed to "QR Code" yet.
